jobs/resource_job.go

```diff
@@ -663,6 +765,15 @@ func ResourceJob() *schema.Resource {
 			if alwaysRunning && js.MaxConcurrentRuns > 1 {
 				return fmt.Errorf("`always_running` must be specified only with `max_concurrent_runs = 1`")
 			}
+			controlRunState := d.Get("control_run_state").(bool)
+			if controlRunState {
+				if js.MaxConcurrentRuns > 1 {
+					return fmt.Errorf("`control_run_state` must be specified only with `max_concurrent_runs = 1`")
+				}
```

Comment:
*nit*: It might be a slightly confusing error message there, as for continuous jobs, `max_concurrent_runs` is set to one and is not changeable. 
I think it would be better to put it after the message about continuous being required.